packages/iov-bcp/src/connection.ts

```diff
@@ -168,8 +168,10 @@ export interface BcpQueryTag {
 
 export interface BcpTxQuery {
   readonly id?: TransactionId;
-  /** send transaction to or from this address */
+  /** any transaction sent to or from this address */
```

Comment:
Hmm, does this change help? What I wanted to say is:

> any SendTransaction with this address as the recipient or the sender

Reply:
Ah yeah, I changed this before we talked about the transaction types. I'll adjust.